src/Shared/AssemblyNameExtension.cs

```diff
@@ -230,15 +230,7 @@ internal static AssemblyNameExtension GetAssemblyNameEx(string path)
                     }
                 }
 
-                var metadataReader = peFile.GetMetadataReader();
-                var entry = metadataReader.GetAssemblyDefinition();
-
-                assemblyName = new AssemblyName();
-                assemblyName.Name = metadataReader.GetString(entry.Name);
-                assemblyName.Version = entry.Version;
-                assemblyName.CultureName = metadataReader.GetString(entry.Culture);
-                assemblyName.SetPublicKey(metadataReader.GetBlobBytes(entry.PublicKey));
-                assemblyName.Flags = (AssemblyNameFlags)(int)entry.Flags;
+                assemblyName = AssemblyName.GetAssemblyName(path);
```

Comment:
Looks like you could collapse the whole `#if` back to the full-framework case (since the API we needed was added in . . . 🥁 . . . .NET Core 2.0).

Reply:
Works for me! I agree it would've felt wrong to have #if do x #else do x 😁